front/src/modules/ui/object/field/types/guards/isFieldSelectValue.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+import { mainColors, ThemeColor } from '@/ui/theme/constants/colors';
+
+const selectColors = Object.keys(mainColors) as [ThemeColor, ...ThemeColor[]];
+const selectValueSchema = z.object({
+  color: z.enum(selectColors),
```

Comment:
I think we want the selectValue to only be a string. The color should be part of the metadata